AsyncPoco.Tests/Program.cs

```diff
@@ -1,15 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿using NUnitLite;
 
 namespace AsyncPoco.Tests
 {
-	public class Program
-	{
-		public static void Main(string[] args)
-		{
-			PetaTest.Runner.RunMainAsync(args).Wait();
-		}
-	}
-}
+    public class Program
+    {
+        /// <summary>
+        /// The main program executes the tests. Output may be routed to
+        /// various locations, depending on the arguments passed.
+        /// </summary>
+        /// <remarks>Run with --help for a full list of arguments supported</remarks>
+        public static int Main(string[] args)
+        {
+            return new AutoRun().Execute(args);
```

Comment:
So this is where NUnitLite CLI comes into play. So if you decide you want to run all the tests from the CLI, then this is how it gets done. Behaves very much like before.